src/lib/components/EventSchedule/index.js

```diff
@@ -26,6 +28,10 @@ import '../EventScheduleModal';
 class EventSchedule extends HTMLElement {
   constructor() {
     super();
+    this.onStateChanged = this.onStateChanged.bind(this);
+
+    this._activeEventDay = null;
+    this._tabsElement = this.querySelector('web-tabs');
```

Comment:
You probably want to do this (and really all DOM operations) in the connectedCallback.